_includes/head.html

```diff
@@ -3,6 +3,9 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   {%- seo -%}
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Metamorphous&display=swap" rel="stylesheet">
```

Comment:
👍 all looking as expected, though I don’t see the font in use in any text yet so can’t say for sure.

It’d be nice to load the font files directly from our site rather than rely on Google Fonts due to the privacy implications, but I think that can wait.